frontend/.gitignore

```diff
@@ -1,3 +1,4 @@
 !.env
 !src/components/config.js
 build/
+!radis.d.ts
```

Comment:
We don't need this. We exclude the ignoring of this file in the parent gitignore. In addition, this isn't the path it's on (it's under `src/`).